lib/src/tools/format_tool.dart

```diff
@@ -464,26 +508,36 @@ FormatExecution buildExecution(
         '${inputs.hiddenDirectories.join('\n  ')}');
   }
 
-  final dartfmt = buildProcess(formatter);
-  final args = buildArgs(dartfmt.args, mode,
-      argResults: context.argResults,
-      configuredFormatterArgs: configuredFormatterArgs);
-  logCommand(dartfmt.executable, inputs.includedFiles, args,
+  final dartFormatter = buildProcess(formatter);
+  Iterable<String> args;
+  if (formatter == Formatter.dartFormat) {
+    args = buildArgsForDartFormat(dartFormatter.args, mode,
+        argResults: context.argResults,
+        configuredFormatterArgs: configuredFormatterArgs);
+  } else {
+    args = buildArgs(dartFormatter.args, mode,
+        argResults: context.argResults,
+        configuredFormatterArgs: configuredFormatterArgs);
```

Comment:
Thoughts on utilizing `buildArgs` for both `dartfmt` & `dart format`? I assume eventually we're going to want to eliminate the use of `dartfmt` so I thought seperating the logic would make that transition easier/ the unit tests are also seperated.

Reply:
Yeah I think this is a good compromise. Also, it looks like [even `dart_style:format` will eventually move to the same CLI](https://github.com/dart-lang/dart_style/issues/986) that `dart format` has now, so eventually we can use that new logic for both.